juniper/src/lib.rs

```diff
@@ -188,6 +189,29 @@ pub enum GraphQLError<'a> {
     IsSubscription,
 }
 
+impl<'a> fmt::Display for GraphQLError<'a> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self {
+            GraphQLError::ParseError(error) => write!(f, "{}", error),
+            GraphQLError::ValidationError(errors) => {
+                let msgs = errors
```

Comment:
I'd rewrite this code to:
```rust
for err in &errors {
    writeln!(f, "{}", err)?;
}
Ok(())
```
It is shorter and avoids allocations.

Reply:
Good call. b7eb36a858217